.github/workflows/ci_workflows.yml

```
@@ -108,3 +108,35 @@ jobs:
       run: python -m pip install --upgrade tox codecov
     - name: Run tests
       run: tox ${{ matrix.toxargs }} -e ${{ matrix.toxenv }} -- ${{ matrix.toxposargs }}
+
+  parallel_and_32bit:
+    name: 32-bit and parallel
+    runs-on: ubuntu-latest
+    container:
+      image: quay.io/pypa/manylinux1_i686
+    steps:
+    # TODO: Use newer checkout actions when https://github.com/actions/checkout/issues/334 fixed
+    - name: Checkout code
+      uses: actions/checkout@v1
+      with:
+        fetch-depth: 0
+    - name: Write configuration items to standard location to make sure they are ignored in parallel mode
+      run: |
+        mkdir -p $HOME/.astropy/config/
+        printf "unicode_output = True\nmax_width = 500" > $HOME/.astropy/config/astropy.cfg
+    # In addition to testing 32-bit, we also use the 3.7 builds to
+    # test the ability to run the test suite in parallel.
+    - name: Install dependencies for Python 3.7
+      run: /opt/python/cp37-cp37m/bin/pip install tox
```

Question: Does

```suggestion
      run: python3.7 -m pip install tox
```

work instead? (Same below with Python 3.8) or are these paths not in `$PATH` by default?

---

Oh, I see! It's a `quay.io/pypa/manylinux1_i686` thing :+1: